src/briefcase/commands/convert.py

```diff
@@ -603,7 +620,16 @@ def build_gui_context(
         # already is set up for a GUI-framework, then those dependencies should already be listed.
         # To prevent the same dependency being listed twice (once in the PEP621-section and once in the
         # briefcase-section), possibly with different versions, we set the GUI-framework to None here.
-        return {"gui_framework": "None"}
+        if self.input_console_app(
+            override_value=project_overrides.pop("console_app", None)
+        ):
+            console_app = "true"
+        else:
+            console_app = "false"
```

Comment:
Matching the changes I made to the template - should this be a string, or a boolean? I suspect it should be a boolean. `input_console_app` is already outputting a boolean.

Reply:
I think, it should be string since making `console_app` a boolean renders `console_app` in `pyproject.toml` to `True` instead of `true`, or is it case insensitive?

Reply:
TOML parsing is case sensitive; I'd argue this means the template should use `{{ "True" if console_app else "False" }}`. The template context should be "pure" data; the template then renders that context as necessary to produce valid TOML.